app/tests/spec/lib/url.js

```diff
@@ -142,6 +142,17 @@ function (chai, _, Url) {
         assert.equal(Url.removeParamFromSearchString('foo', ''), '');
       });
     });
+
+    describe('isNavigable', function () {
+      it('detects HTTP and HTTPS properly', function () {
+        assert.isFalse(Url.isNavigable('urn:ietf:wg:oauth:2.0:fx:webchannel'));
+        assert.isTrue(Url.isNavigable('https://find.firefox.com'));
+        assert.isTrue(Url.isNavigable('http://find.firefox.com'));
+        assert.isFalse(Url.isNavigable(''));
+        assert.isFalse(Url.isNavigable());
```

Comment:
A brief look at the code - it also looks like both the strings `http://` and `https://` (without any sort of domain name) are considered navigable. I think they should not be.